landing-page/styles.css

```diff
@@ -0,0 +1,122 @@
+*{
+  margin:0;
+  padding:0;
+}
+body{
+  margin:0;
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+  font-size: 17px;
+  line-height:1.6;
+}
+#simbol {
+  text-align: center;
+  color: black;
+  font: italic small-caps bold 30px Georgia, Garamond, serif;
+  background: limegreen;
+  margin: 10px;
+  border: 7px solid purple;
+  padding: 10px;
+
+}
+#showcase {
+  background-image:url('https://www.designcuts.com/wp-content/uploads/2018/11/30-Spring-Watercolor-Backgrounds-strip5.jpg');
+  background-size: cover;
+  background-position: center;
+  height:100vh;
+  display:flex;
+  flex-direction:column;
+  justify-content:center;
+  align-items:center;
+  text-align:center;
+  padding:0 20px;
+}
+#showcase h1 {
+  font-size:30px;
+  line-height:1.2;
+  }
```

Comment:
brace is indented here -- these are easy to miss. Also, you should decide if you are going to leave a blank line between each set of rules or not -- I like having that blank line because it makes things easier to read. But some people don't -- whatever you decide, just keep it consistent.